### PR TITLE
Small LocalDcm2JsonOperator cleanups

### PR DESCRIPTION
**Pull request by hmeine**
I have been reading the code of LocalDcm2JsonOperator and made a few cleanups along the way.

### CODE REVIEW

File: data-processing/kaapana-plugin/extension/docker/files/plugin/kaapana/operators/LocalDcm2JsonOperator.py [hunk 1]
```diff
@@ -290,6 +290,7 @@ def replace_tags(self, dicom_meta):
                 vr = str(value['vr'])
 
                 if "nan" in value_str:
+                    # FIXME: doesn't this cause false positives, e.g. if "nanotech" appears in some string tag?
```
Comment (hmeine) [inline data-processing/kaapana-plugin/extension/docker/files/plugin/kaapana/operators/LocalDcm2JsonOperator.py:291]: I realized the probability for wrong matches is very low: `value_str` is still a *list* at this point, so 'nan' would have to appear as a single element in that list. (The conversion to a string is immediately below.)

**Comment by pschader** [comment]
Dear @hmeine thank you very much for your pull request. We currently creating contribution guidelines to make including external contributions easier and faster. For now, to accept your contribution we would need you to post the appended declaration in this discussion. Sorry for the inconvenience.

Here the declaration:
```
By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
are public and that a record of the contribution (including all
personal information I submit with it, including my sign-off) is
maintained indefinitely and may be redistributed consistent with
this project or the open source license(s) involved.
```

**Comment by hmeine** [comment]
Sure!

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
are public and that a record of the contribution (including all
personal information I submit with it, including my sign-off) is
maintained indefinitely and may be redistributed consistent with
this project or the open source license(s) involved.

**Comment by pschader** [comment]
It took us a while but finally your changes are now merged and will be part of the upcoming release.

We rebased your commits on develop leading to different commit hashes and therefore this pull request is not closed automatically but [your code](https://codebase.helmholtz.cloud/kaapana/kaapana/-/commit/572fa1626497410232dfc698b1cfd9d821d9aed1) is in the codebase now and you are also listed under the [contributors](https://github.com/kaapana/kaapana/graphs/contributors).

Thanks a lot for your contribution!

**Pull request closed without merging**